frontend/src/js/components/tenants/TenantsForm.test.tsx

```diff
@@ -47,14 +47,16 @@ describe('TenantsForm', () => {
     await user.type(screen.getByPlaceholderText('Password'), newChildTenant.password);
     await user.type(screen.getByLabelText('Set device limit'), `{backspace}${newChildTenant.dev}`);
     await user.click(screen.getByText(/enable delta artifact generation/i));
+    await user.click(screen.getByText(/reset the password/i));
     await user.click(screen.getByRole('button', { name: /Create Tenant/i }));
 
     expect(submitTenantSpy).toHaveBeenCalledWith({
       admin: { email: newChildTenant.email, password: newChildTenant.password },
       name: newChildTenant.name,
       device_limit: Number(newChildTenant.dev),
       binary_delta: true,
-      sso: false
+      sso: false,
+      send_reset_password: true
```

Comment:
It seems like the send_reset_password should be inside the `admin` object.
https://github.com/mendersoftware/mender-server-enterprise/blob/094f21433dfb7bfa83fd443c12161882640a24ec/backend/services/tenantadm/docs/management_api_v2.yml#L1322-L1337

Reply:
Otherwise I can confirm the behavior works :+1: